.eslintrc

```diff
@@ -42,6 +42,7 @@
     "promise/always-return": "error",
     "promise/catch-or-return": "error",
     "promise/no-native": "off",
+    "quotes": ["error", "single", { "avoidEscape": true }],
```

Comment:
Please explain why you made this change

Reply:
Removed, but this was to be able to test since prettier is being used with single quote.